benchmarks/distributed/rpc/rl/plot_benchmark_report.py

```diff
@@ -0,0 +1,33 @@
+import json
+import matplotlib.pyplot as plt
+import numpy as np
+
+with open('report.json') as report:
+        report = json.load(report)
+
+x_axis_name = report['x_axis_name']
+bar_width = 0.35  # the bar_width of the bars
+
+benchmark_results = report['benchmark_results']
+x_axis_labels = [benchmark_run[x_axis_name] for benchmark_run in benchmark_results]
+label_location = np.arange(len(x_axis_labels))
+
+for benchmark_metric in ['agent throughput', 'observer throughput', 'agent latency', 'observer latency']:
```

Comment:
each benchmark result has the x variable name/value as a key value pair in addition to the 4 metrics (for example, batch: True), I thought this useful since users may want to see in the report itself what x value the metrics correspond to.  I can easily change the format slightly so that I can call keys() on a dictionary and only get those 4 metrics, however I figured the way its currently set up helps with readability for someone less familiar with the code.  I trust your experience over mine, if you think its better practice to call keys() on a dictionary let me know

Reply:
I see. Yep, sure, let's keep it.